README.md

```diff
@@ -56,6 +54,44 @@ brew install foundry
 
 Run `foundry version` to verify it's installed correctly.
 
+## Getting Started
+
+Clone this sample to your local system, or [download as a zip file](https://github.com/CrowdStrike/foundry-sample-scalable-rtr/archive/refs/heads/main.zip).
+
+```shell
+git clone https://github.com/mraible/foundry-sample-scalable-rtr
+cd foundry-sample-scalable-rtr
+```
+
+Log in to your Foundry instance:
+
+```shell
+foundry login
+```
+
+Select the following permissions:
+
+- [x] Run RTR Scripts
+- [x] Run, execute, and test Workflows
+- [x] Run, execute, and test API integrations
+- [x] Run, execute, and test LogScale queries
+- [ ] (optional) Generate mock data to test your app
+
+Deploy it to Foundry:
+
+```shell
+foundry apps deploy
+```
+
+> [!TIP]
+> If you get an error that the name already exists, change the name to something unique to your CID in `manifest.yml`.
+
+Once the deployment has finished, release the app using the three-dot menu on the right. Next, go to **Foundry** > **App catalog**, find your app, and install it.
+
+Then, open the main menu in the top left and go to **Custom Apps**. You should see your app listed and be able to enter a job. After the job executes, you'll receive an email with the execution information.
+
+## Basic information
```

Comment:
How about `About this app` instead of this somewhat vague heading?

Reply:
I'm OK with this change. This was an original header that I didn't modify. My only question is which is better:

- About this app
- About this sample 
- About this sample app

Reply:
got it, I wasn't involved with the original structure either - I like "About this sample app"